### PR TITLE
Leaderboard Ranking Proposal Draft

### PR DESCRIPTION
**Pull request by nilsleh**
This PR adds a new Ranking Results page, based on all_results.csv. I tried to highlight different methods of ranking aggregation that are commonly found in other comprehensive benchmark. In particular this proposal is largely inspired by [TabArena](https://huggingface.co/spaces/TabArena/leaderboard) which is a quite comprehensive and accepted benchmark for ML Tabular benching. It includes different ranking aggregations:

- Average Rank
- ELO
- Mean Improvability

I used [evaluma](https://github.com/nilsleh/evaluma) for these ranking things. The rankings are precomputed and stored and then rendered in the static html.

As part of this PR I also found some duplicate rows in the all_results.csv file, as well as a lot of incomplete model/dataset combinations. Therefore, the rendered ranking html page should not be used to draw conclusions yet, but merely as an example of what a provided ranking UI could look like and how to present ranking results that people can intuitively explore.

EDIT: Live preview made available by Konstantin https://raw.githack.com/torchgeo/torchgeo-bench/leaderboard_rankings/docs/_static/ranking_explorer.html

### CODE REVIEW

**Comment by nilsleh** [comment]
Snapshot of the leaderboard. Otherwise you can look at it in your local browser by pasting: torchgeo-bench/docs/_static/ranking_explorer.html

Screenshot from 2026-07-09 11-32-11

**Comment by calebrob6** [comment]
I like it!! I'll resolve the all_results conflict

**Comment by nilsleh** [comment]
Screenshot from 2026-07-28 11-10-02

added performance profiles

**Comment by hannah-rae** [comment]
Awesome @nilsleh! Here are my comments - I would consider all of these extensions to what you have now (and each of these have more decisions/planning in themselves), so I can make issues for them to add on later if yall think they are worth pursuing.
- Error bars: I know we've discussed this in meetings before but can't remember what we landed on. Do we want them accounted for in the ranks, or just assume closeness will be taken care of by other metrics?
- Page organization: We talked in the last meeting about how different questions would be answered by different tables/plots on this page. What if we put something like an FAQ list or "I care about..." list at the top that people can use to jump to the results that will be most meaningful to them? 
- TL;DR: I think we should add one. Maybe like a short paragraph abstract or bulleted list that summarizes everything at the top. 
- Categories: I would love to add "application areas" to group the datasets and get ranks within each of those (could be a dropdown or checklist beside the current filters) e.g.: agriculture, disasters, water, forestry, urban/land use, infrastructure, maritime, defense/security, climate/carbon/energy, biodiversity/ecosystems. 
- One rank to rule them all: Currently there are different ranks depending on which settings you select for task, probe, etc. I think we should consider having one table that combines all of them. Basically regardless of which options you choose, how do the models rank. (This is kind of answered by Rank Sensitivity I think, but not exactly the same or as straightforward to understand)
- I think we can improve the Dolan–Moré profiles description. IMHO it is really confusing even for someone who is immersed in research on this topic, so for non-experts I think it will be really hard to follow. (Actually a lot of the descriptions make vague references that are hard to follow, like to "the table" or "right dropdown")
- The "INSIGHTS" header is weird: are the results before not insights..?
- I find the rank flow really difficult to read. Also I don't think it will work if/when we have more than 2 options for each condition family, since you won't be able to flip (though I guess we could choose the worst rank or sort them and have multiple lines). Some things I think would help the current plot: hovering over a line could highlight the whole line (start and end rank), the popup could give more information like what was changed and what the start/end rank is (I think this is somewhat supposed to show in the box beneath it but I also find that box hard to follow. It's also weird when there are models that don't have multiple options, like for Bands the first four don't have a flip so they disappear from the right-side list.
- Rank dispersion: maybe we can add colors/annotations to the quadrants to make it easier to see on the plot rather than referring to the descriptions. Same comment as for rank flow, I think it's kind of weird to have this separate box of details.
- Performance vs FLOPs: can we add a scatter plot of this? We could make each model a circle or points connected by a line that accounts for the different settings.

**Comment by LeWaldm** [comment]
As mentioned in #191 :

Would it make sense to set the **default of Normalization to "Model-native"?** To me, this makes most sense as it seems closest to how the models were pre-trained and intended to be used downstream. Panopticon, for example, performs poorly on m-forestnet and m-so2sat with "Band-spec z-score" normalization but obtains the expected scores (reported in the paper) with "Model-native" normalization.
